src/web_ui/templates/realtime_readings.html

```diff
@@ -58,7 +58,7 @@ <h1 class="h3 mb-3 text-center">Readings result {% if node_id %} for {{ node_id
                           <td class="w-50">
                             <span class="ml-5">{{ item['description'] or item['var'] }}</span>
                           </td>
-                          <td class="w-50">{{ item['value'] }} {{ item['unit'] }}</td>
+                          <td class="w-50">{{ item['value'] if item['value'] or 'No data' }} {{ item['unit'] if item['value'] is not none else '' }}</td>
```

Comment:
I suspect that this will show "No data" if `item['value']` is 0, which is not what we want. I'm not sure of the syntax, but we should display `No data` only where the value is actually `None`/`null`